server/helpers/projectsChecker.js

```diff
@@ -34,7 +34,7 @@ const getRepoListWithCommits = async urls => {
   const repoList = urls.split(',').map(url => url.replace('https://github.com/', '').trim());
   const commitsMap = {};
   const promises = repoList.map(async repo => {
-    commitsMap[repo.split('/commits')[0]] = await getCommits(repo);
+    commitsMap[repo] = await getCommits(repo);
```

Comment:
update keys to be full url since all students in the same group will have the same repo reference